Library/Formula/dbxml.rb

```diff
@@ -0,0 +1,27 @@
+require 'formula'
+
+class Dbxml < Formula
+  url 'http://download.oracle.com/berkeley-db/dbxml-6.0.17.tar.gz'
+  homepage 'http://www.oracle.com/us/products/database/berkeley-db/xml/overview/index.html'
+  sha1 'fa706a73497bbdc6e026cd83e8a5538dd9468e0c'
+
+  depends_on 'xerces-c'
+  depends_on 'xqilla'
+  depends_on 'berkeley-db'
+
+  def install
+    inreplace "dbxml/configure" do |s|
+      s.gsub! "lib/libdb-*.la | sed 's\/.*db-\\\(.*\\\).la", "lib/libdb-*.a | sed 's/.*db-\\(.*\\).a"
+      s.gsub! "lib/libdb-*.la", "lib/libdb-*.a"
+    end
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!